index.bs

```diff
@@ -2888,6 +3055,8 @@ a [=moment=] |triggerTime|, and a [=boolean=] |fenced|:
         return null.
     1. If |aggregatableSourceRegTimeConfig| is not "<code>[=aggregatable source registration time configuration/exclude=]</code>", return null.
     1. Set |triggerContextID| to |value|["<code>[=trigger-registration JSON key/trigger_context_id=]</code>"].
+1. Let |attributionScopes| be the result of running [=parse attribution scopes=] with |value| and [=max number of attribution scopes=].
+1. If |attributionScopes| is not null and [=set/is empty|empty=], return an error.
```

Comment:
This algorithm never returns null, so something is not right here.

Reply:
Done.